bin/good-first-issue.js

```diff
@@ -13,6 +15,14 @@ cli
   .description('CLI tool to find good first issues.')
   .arguments('<project>')
   .action((project) => {
+    // if project is not found
+    if (!(project in projects)) {
+      console.log(chalk.red(`${project} was not found in good-first-issue`))
+      console.log('--------------------------------')
+      console.log('If you\'d like to add a new project to good-first-issue')
+      console.log('please read - https://github.com/bnb/good-first-issue#adding-new-projects')
```

Comment:
Add color to the link 😁 

```suggestion
      console.log('please see the module's GitHub repository:' + chalk.cyan('https://github.com/bnb/good-first-issue#adding-new-projects'))
```